voluncheer/job_board/models.py

```diff
@@ -2,9 +2,11 @@
 
 
 class Job(models.Model):
-    organization = models.ForeignKey("profiles.Organization", on_delete=models.CASCADE)
+    organization = models.ForeignKey(
+        "profiles.Organization", on_delete=models.CASCADE
+    )  # noqa: E501
```

Comment:
We should really try hard to avoid adding in exceptions to the linter. There is almost always a way to make it work. If you need help figuring out how to make it fit the style preference just let me know and we can figure it out.

Here and elsewhere.

Reply:
Sure thing, the wrap is also done by autocorrect on vs code. The # noqa: E501 I added is for flask8 error ignoration.